### PR TITLE
VIH-6620 Remove Questionnaire for Rep

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-6620

### Change description ###
Remove questionnaire for representative

**Does this PR introduce a breaking change?** (check one with "x")

```
[X ] Yes
[ ] No
```

### CODE REVIEW

**Review by RocheL — DISMISSED** [review]

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Pages/Journeys/RepresentativeJourney.cs [hunk 1]
```diff
@@ -33,6 +29,8 @@ public void VerifyUserIsApplicableToJourney(UserType userType)
 
         public void VerifyDestinationIsInThatJourney(Page destinationPage)
         {
+            var jj = Journey().Contains(destinationPage);
+
```
Comment (npoxon) [inline ServiceWebsite/ServiceWebsite.AcceptanceTests/Pages/Journeys/RepresentativeJourney.cs:33]: should this be here?

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Pages/Page.cs [hunk 2]
```diff
@@ -47,6 +32,8 @@ private Page(string name, string url)
         public static readonly Page EquipmentBlocked = new Page("Equipment Blocked", "equipment-blocked");
 
         // Shared
+        public static readonly Page CheckingVideoHearing = new Page("Checking Video Hearing", "checking-video-hearing");
+
```
Comment (npoxon) [inline ServiceWebsite/ServiceWebsite.AcceptanceTests/Pages/Page.cs:36]: Is this a shared page or just Rep? You have only added a test for Rep if so?

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Pages/Journeys/RepresentativeJourney.cs (hunk 1 above)
Comment (valentinabennett) [reply ServiceWebsite/ServiceWebsite.AcceptanceTests/Pages/Journeys/RepresentativeJourney.cs:33]: No it should not. Missed to remove.

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Pages/Page.cs (hunk 2 above)
Comment (valentinabennett) [reply ServiceWebsite/ServiceWebsite.AcceptanceTests/Pages/Page.cs:36]: It's shared page.

File: ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Individual/CheckingVideoHearing.feature [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 	So that the court can make an informed decision based on my circumstances
 
 @Individual
-Scenario: About Hearings
-	Given the Individual has progressed to the About Hearings page
+Scenario: Checking video Hearing
+	Given the Individual has progressed to the Checking Video Hearing page
 	Then contact details are available
 	When the user clicks the Next button
 	Then the user is on the Check Your Computer page
```
Comment (npoxon) [inline ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Individual/CheckingVideoHearing.feature:11]: Can you put this into the ../Shared directory and add the Individual and Rep tests to the same file?

Comment (valentinabennett) [reply ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/Individual/CheckingVideoHearing.feature]: I moved it to shared features

**Review by npoxon — APPROVED** [review]

**Pull request merged**